src/facade/facade.cc

```diff
@@ -51,7 +51,9 @@ ConnectionStats& ConnectionStats::operator+=(const ConnectionStats& o) {
 #undef ADD
 
 string WrongNumArgsError(string_view cmd) {
-  return absl::StrCat("wrong number of arguments for '", cmd, "' command");
+  string lcmd(cmd);
+  absl::AsciiStrToLower(&lcmd);
```

Comment:
Abseil already has a one-liner for your use case https://github.com/abseil/abseil-cpp/blob/2fc358dab079c220feb81f75b5a366f8d1464cd4/absl/strings/ascii.h#L170-L175

Reply:
Oh I haven't seem that. Fixed!
And also changed lowered the command on the InvalidExpireTime